fdbserver/storageserver.actor.cpp

```diff
@@ -1947,11 +1982,23 @@ ACTOR Future<Void> logFetchKeysWarning(AddingShard* shard) {
 }
 
 ACTOR Future<Void> fetchKeys( StorageServer *data, AddingShard* shard ) {
+	state UID ID = deterministicRandom()->randomUniqueID();
 	state TraceInterval interval("FetchKeys");
 	state KeyRange keys = shard->keys;
 	state Future<Void> warningLogger = logFetchKeysWarning(shard);
-	state double startt = now();
+	state const double START_TIME = now();
```

Comment:
We use lowerCamelCase for local variables (const or not).

ALL_CAPS is reserved for compile time constants (roughly, things that could be `constexpr`), though SERVER_KNOBS is an obvious exception.